packages/client/src/pages/BusinessDashboard.tsx

```
@@ -0,0 +1,210 @@
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string | null>(null);
+  const [isSubmittingContract, setIsSubmittingContract] = useState<boolean>(false);
+
+  useEffect(() => {
+    
+    const fetchContractors = async () => {
+      setIsLoading(true);
+      setError(null);
+      try {
+        const response = await getAllContractors();
+        if (response && response.contractors) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)